### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by developer-guy**
1 - In stack.YAML, I saw that these functions use golang-http template, so when I followed the documentation I got an error because of the absence of the template, so fixed that problem by adding template pull command.
Screen Shot 2020-05-01 at 00 18 57

2 - When I tried to apply connector-dep manifest, I got an error too, I saw this file prefixed with yaml not yml, so I fixed that too.
Screen Shot 2020-05-01 at 00 27 37

and here is the final result of the nats-connector demo, congrats it's working 😍 
Screen Shot 2020-05-01 at 00 30 56

### CODE REVIEW

**Comment by alexellis** [comment]
`faas-cli template store pull golang-http` shouldn't be necessary if the stack.yml file contains the list of templates required.

Have a look at : https://docs.openfaas.com/reference/yaml/#yaml-template-stack-configuration

**Comment by developer-guy** [comment]
> `faas-cli template store pull golang-http` shouldn't be necessary if the stack.yml file contains the list of templates required.
> 
> 
> 
> Have a look at : https://docs.openfaas.com/reference/yaml/#yaml-template-stack-configuration

Ok then we should add this command below :

$ faas-cli pull template stack

Right?

**Review by LucasRoesler — APPROVED** [review]
Looks great to me, thanks 💪

**Comment by alexellis** [comment]
No, no pull command should be required at all once you've updated the YAML file to list the template from the store or a URL. @martindekov can confirm. I can merge this, but it's not meant to be necessary at all.

File: README.md [hunk 1]
```diff
@@ -45,7 +46,8 @@ The following instructions show how to run and test `nats-connector` on Kubernet
 
       ```
    cd contrib/test-functions
-   faas-cli deploy --filter receive-message
+   faas-cli template pull stack
+   faas-cli deploy --filter publish-message
```
Comment (LucasRoesler) [inline README.md:50]: I think we can also do this
```suggestion
   faas-cli deploy --filter publish-message --read-template=false
```

File: README.md [hunk 2]
```diff
@@ -26,13 +26,14 @@ The following instructions show how to run and test `nats-connector` on Kubernet
    Or deploy with the `stack.yml` provided in this repo:
    ```
    cd contrib/test-functions
+   faas-cli template pull stack
    faas-cli deploy --filter receive-message
```
Comment (LucasRoesler) [inline README.md:30]:
```suggestion
   faas-cli deploy --filter receive-message --read-template=false
```

Comment (developer-guy) [reply README.md:30]: i updated @LucasRoesler

File: README.md (hunk 1 above)
Comment (developer-guy) [reply README.md:50]: i updated @LucasRoesler

**Review by alexellis — APPROVED** [review]
LGTM

**Pull request merged**